src/components/tooltip/tooltip.js

```diff
@@ -106,9 +106,12 @@ function MdTooltipDirective($timeout, $window, $$rAF, $document, $mdUtil, $mdThe
      */
     function getParentWithPointerEvents () {
       var parent = element.parent();
-      while (parent && hasComputedStyleValue('pointer-events','none', parent[0])) {
+
+      // jqLite might return a non-null, but still empty, parent; so check for parent and length
+      while (parent && parent.length && hasComputedStyleValue('pointer-events','none', parent[0])) {
```

Comment:
@topherfangio - I think we should move the **hasComputedStyleValue** to `$mdUtil.hasComputedStyleValue( )`; which would simplify testing of this feature.